internal/charmstore/store.go

```diff
@@ -895,6 +895,14 @@ func (s *Store) findZipFile(blob io.ReadSeeker, size int64, isFile func(f *zip.F
 	return mongodoc.ZipFile{}, params.ErrNotFound
 }
 
+// SetPerms sets the permissions for which ("read" or "write")
+// to the given ACL.
+func (s *Store) SetPerms(id *charm.Reference, which string, acl ...string) error {
+	return s.DB.BaseEntities().UpdateId("cs:~charmers/wordpress", bson.D{{"$set",
+		bson.D{{"acls." + which, acl}},
+	}})
+}
```

Comment:
I'm lost a bit. Why is this in store.go and it updates only cs:~charmers/wordpress?